pages/docs/tutorials/kafka/index.md

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
do you need this?

Reply:
I assume so. It shows the start and end of the metadata. And that's whats across the docs. Except I am missing something.

Reply:
sorry, I meant the file not `---`. I guess you have it just for the sake of PR preview?

Reply:
best check with other docs contributors for Kafka,
not sure maybe you agreed you need this file

Reply:
The index.md and section.md files are default files for folders created for documentation.

Reply:
Heyo @CynthiaPeter, since we will merge Joy's tutorials first because they are the start of our series, then your PRs do not need to include this `_section.md` or `index.md` files. You can remove these from your PR and keep just your tutorial. 

let me know if that makes sense 😸

Reply:
Done.